web/pdf_page_view.js

```diff
@@ -189,6 +197,17 @@ var PDFPageView = (function PDFPageViewClosure() {
         rotation: totalRotation
       });
 
+      if (this.svg) {
+        this.cssTransform(this.svg, true);
```

Comment:
Nit: With `cssTransform` now being used with SVG rendering as well, it's a bit confusing that the method signature is still
`cssTransform: function PDFPageView_transform(canvas, redrawAnnotations) {`.

I thus think that it would be clearer if the `canvas` parameter was changed to e.g. `renderTarget` (or similar), so the method signature instead becomes:
```javascript
cssTransform: function PDFPageView_transform(renderTarget,
                                             redrawAnnotations) {
```